tox.ini

```diff
@@ -30,6 +29,8 @@ setenv =
     TOX_ENVBINDIR={envbindir}
     TOX_NRNBINDIR=../.tox-neuronpy/local/x86_64/bin/
 commands =
+    git clone -b optimization https://github.com/russelljjarvis/neuronunit
```

Comment:
Can't we make pip install this automatically?

Reply:
Yes.